doc_build/BUILD

```diff
@@ -11,16 +11,39 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Generate the reference documentation.
+
+How to:
+   bazel build //doc_build:reference
+   cp bazel-bin/doc_build/reference.md docs/latest.md
+   git commit -m 'update docs' docs/latest.md
```

Comment:
It would be nice if we could `bazel run` this as one command, but not a critical matter.

Reply:
I could. Maybe some day. Perhaps a pkg_install command eventually.